src/environments/environment.ts

```diff
@@ -6,7 +6,8 @@ export const environment = {
   production: false,
   // host: 'http://wp-np3-e3:9200/'
   // host: 'http://ves-hx-e4:9200/faang_build_2/'
-  host: 'http://data.faang.org/api/data/'
+  host: 'https://data.faang.org/api/',
```

Comment:
Please revert this change as /data endpoint would work when the API is moved to EC v4